### ISSUE TITLE
HiDPI seems broken

### ISSUE THREAD

**Issue by rikai** [djfun/audio-visualizer-python#9]
At least under HiDPI gnome, the text size in the preview looks pretty messed up:
![image](http://i.imgur.com/8gPNTKB.png)

And that continues into the exported product:
![image](http://i.imgur.com/rZvApEq.jpg)

My current working theory is that UI scaling is messing with the font size, waiting on someone with HiDPI to try in a non-scaled environment to see if it corrects the issue and if so we can continue from there. Could also be something else, though.

The UI looks a bit messy as well, but that's somewhat expected, tbh. It looks about as bad on the mac, so that's a separate issue.

**Comment by djfun** [comment]
is this still a problem? if so, we might need to change to qt5, which has better High-DPI support, to fix it.
also: what priority does this have in respect to #6 and #7?

**Comment by rikai** [comment]
This is still a problem as far as i'm aware.

I'd say this probably has the highest priority as it makes the program unusable in certain use cases, followed by #7 and then #6 

Unrelated, but i wasn't sure where else to ask. I was also speaking to some other interesting folks just today about expanding the capabilities of this project in some interesting ways. One idea that was had was to write some python bindings for [Project M](https://github.com/projectM-visualizer/projectm) and use it to allow for much, much more varied visualization options to layer over images. Would you be open to an idea like this?

**Comment by djfun** [comment]
ok, I'll look into fixing this issue.

I am interested in providing different visualizations with this project. I see a challenge in using the real-time approach of Project M in a way that is compatible to the frame-by-frame method currently used. Also I don't know if I can do that all myself, but I am glad to help if there are others who support that task.

**Comment by djfun** [comment]
I pushed some fixes to https://github.com/djfun/audio-visualizer-python/tree/feature-hdpi
If you can try it (or know someone who can), that would be great.

**Comment by rikai** [comment]
Sadly i don't have access to a HiDPI display atm, so i can't test it myself, but i'll see if i can't find someone to test it.

**Comment by djfun** [comment]
tests show it is working better now with c622f2908ed1e1fed165d3433d7e6932f06c7a22
